### ISSUE TITLE
Tab Bar Navigation

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#143]
The voiceover is not recognizing the Tab Bar properly and is saying Tab description.

It should be saying Selected (Tab bar selected 'publications' ), tab 1 of 4, and rest for unselected.

'Search' tab 2 of 4,

'For you' tab 3 of 4,

'Settings' tab 4 of 4.

**Comment by ForumG-SC** [comment]
Duplicate https://github.com/StatCan/StatcanPublicationUAT/issues/67
